ProjectProposal.md

```diff
@@ -0,0 +1,27 @@
+# What are we building
+We are building a simple web-app that allows users to maintain an online task list. 
+
+## Main features
+- Create, read, update, and delete tasks from the web interface
+- Tasks have a title and a description
+- Tasks can be marked as new, active, or completed
+- Tasks are stored in the cloud and can be accessed via a user account on any device
```

Comment:
Thank you @listonjesse This looks awesome, A question here: Is the '-' here is the same as '*' used in the markdown list?

Reply:
Q2: If I request changes, can I work on the same branch as you to help with the changes

Reply:
'-' and '*' both make a bullet in an unordered list

For question 2, I don't mind if you checkout this branch and push changes to it directly